Intermittent failure in contention_resolver_test

See https://github.com/kaleido-io/paladin/actions/runs/10721217811/job/29729212637?pr=129

```
--- FAIL: TestContentionResolver_2TransactionsDeterministicResults (3.61s)
    contention_resolver_test.go:49: 
        	Error Trace:	/home/runner/work/paladin/paladin/kata/internal/engine/sequencer/contention_resolver_test.go:49
        	Error:      	Max difference between 551 and 449 allowed is 100, but difference was 102
        	Test:       	TestContentionResolver_2TransactionsDeterministicResults
```

--- a/core/go/internal/engine/sequencer/contention_resolver_test.go
+++ b/core/go/internal/engine/sequencer/contention_resolver_test.go
@@ -25,8 +25,8 @@ import (
 )
 
 func TestContentionResolver_2TransactionsDeterministicResults(t *testing.T) {
-	t.Skip("This test has non zero probability of failing")
-	// ... which means it has a high probabiliy of failing at some point in time if we run it on every CI build
+	t.Skip("this test should be run manually when the algorithm is changed. ")
+	// see https://github.com/kaleido-io/paladin/pull/145 for background
 
 	// create 2 ids at random (representing bidding transactions),
 	// then iterate over 100 random state ids and check that there is a fair distribution of winners

